cf/cognito-case-insensitive.yaml

```diff
@@ -129,4 +131,14 @@ Resources:
     Type: AWS::Cognito::UserPoolDomain
     Properties:
       UserPoolId: !Ref UserPool
-      Domain: !Sub "hms-auth-${Environment}-${AWS::AccountId}"
```

Comment:
this being `hms-auth...` is so that when users land on the login page the url doesn't have `biomage-auth...` in it (which is not appropriate for the HMS deployment)

Reply:
I added a variable to store `hms-auth` or `biomage-auth` depending on the deployment in [this PR](https://github.com/biomage-org/iac/pull/206). 

However, In the same file there are other occurances of `biomage`, which have not been renamed in the master branch of this repo. Is this intended?

Reply:
If you are referring to infrastructure names (e.g. `biomage-user-pool-case-insensitive....`), then yes, this is an unfortunate legacy that cannot be easily undone. HMS deployments were originally deployed with the current infrastructure naming conventions and most AWS infrastructure cannot just be renamed.

That said --- **all future infrastructure** should either not contain organization names within them or use a configurable value.